tools/specgen/resttag.go

```diff
@@ -6,13 +6,13 @@ import (
 )
 
 const (
-	Header = "header"
-	Body   = "body"
-	Query  = "query"
+	header = "header"
```

Comment:
Makes golint happy.  The choices were comment them since they were public or make them private.  Since no one outside of this file used it, I made them private.